lib/pf/Switch/Ubiquiti/Unifi.pm

```diff
@@ -245,6 +288,170 @@ sub _deauthenticateMacWithHTTP {
     $logger->info("Switched status on the Unifi controller using command $command");
 }
 
+=item getMACIP
+
+get IP associated to a AP MAC
+
+=cut
+
+
+sub getMACIP {
```

Comment:
can you make the name more explicit like getAccessPointMACIP so that we don't think this can be for a node